plat/drivers/virtio/virtio_net.c

```diff
@@ -830,10 +858,15 @@ static __u16 virtio_net_mtu_get(struct uk_netdev *n)
 	return d->mtu;
 }
 
-static int virtio_netdev_feature_negotiate(struct virtio_net_device *vndev)
+static int virtio_netdev_feature_negotiate(struct uk_netdev *n)
 {
 	__u64 host_features = 0;
+	__u64 drv_features  = 0x0;
```

Comment:
No need to write it in hex if it's 0.

Reply:
Sure not needed, I can remove it if you prefer - I don't mind. I find it supportive to the understanding that we are expecting bit flags to be set when using the `0x` notation - kind of: "this is not an ordinary number, we will do bit-wise operations on it."